unattended_installer/install_functions/dashboard.sh

```diff
@@ -188,3 +188,27 @@ function dashboard_install() {
     fi
 
 }
+
+function dashboard_installReportDependencies() {
+
+    # Flags that indicates that is an optional installation.
+    optional_installation=1
+    report_dependencies=1
+
+    installCommon_checkChromium
+
+    if [ "${sys_type}" == "yum" ]; then
+        dependencies+=( nss xorg-x11-fonts-100dpi xorg-x11-fonts-75dpi xorg-x11-utils xorg-x11-fonts-cyrillic xorg-x11-fonts-Type1 xorg-x11-fonts-misc fontconfig freetype )
```

Comment:
Use a different variable name or make empty the list before setting the values.

Reply:
Done in https://github.com/wazuh/wazuh-packages/pull/2122/commits/614e83b7be2d794699e48f8f16979351918d24f2